src/openvr_plugin/driver_psmoveservice.cpp

```diff
@@ -2103,6 +2103,41 @@ void CPSMoveControllerLatest::UpdateControllerState()
 			{
 				PSMoveButtonState resetPoseButtonState = clientView.GetButtonSelect();
 
+				const bool bHasChildNavi =
+					m_PSMChildControllerView != nullptr &&
+					m_PSMChildControllerView->GetControllerViewType() == ClientControllerView::eControllerType::PSNavi;
+				if (bHasChildNavi)
+				{
+					// Load config from steamvr.vrsettings
+					vr::IVRSettings *pSettings = m_pDriverHost->GetSettings(vr::IVRSettings_Version);

```

Comment:
No worries. Sorry I put it in the wrong place. I've changed it like you said and tested to make sure it still seemed to work the same. I hope there aren't any other problems with it.